src/poc/server/poc-server/src/index.ts

```diff
@@ -5,11 +5,18 @@ dotenv.config();
 
 const app: Express = express();
 const port = process.env.PORT || 3000;
+app.set("view engine","ejs")
+
 
 app.get("/", (req: Request, res: Response) => {
   res.send("Express + TypeScript Server");
 });
 
+app.get("/portal", (req: Request, res: Response) => {
```

Comment:
The server in this case doesn't need to serve HTML. The UI components can be specified in /src/poc/client/poc-client

Reply:
i.e. the UI changes should be /src/poc/client/poc-client